src/Sylius/Behat/Context/Api/Admin/ManagingCountriesContext.php

```diff
@@ -76,23 +81,47 @@ public function iAddIt(): void
      */
     public function iWantToEditThisCountry(CountryInterface $country): void
     {
-        $this->client->buildUpdateRequest(Resources::COUNTRIES, $country->getCode());
+        $showRequestBuilder = $this->requestBuilderFactory->show(
+            PathPrefixes::ADMIN_PREFIX,
+            Resources::COUNTRIES,
+            $country->getCode(),
+            'Authorization',
+            $this->sharedStorage->get('token')
+        );
+        $showResponse = $this->client->executeCustomRequest($showRequestBuilder->build());
```

Comment:
I wonder if we shouldn't stay with:
```suggestion
        $showResponse = $this->client->show(Resources::COUNTRIES, $country->getCode());
```
as the creation of request is there, so the problem with statefulness of this service is solved, but this will reduce the amount of unnecessary code in contexts.

Or
```suggestion
        $showRequest = $this->requestFactory->show(
            PathPrefixes::ADMIN_PREFIX,
            Resources::COUNTRIES,
            $country->getCode(),
            'Authorization',
            $this->sharedStorage->get('token')
        );
        $showResponse = $this->client->executeCustomRequest($showRequest);
```